pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -176,6 +197,46 @@ func (c *MizarArktosNetworkController) syncNetwork(eventKeyWithType KeyWithEvent
 func (c *MizarArktosNetworkController) processNetworkCreation(network *v1.Network, eventKeyWithType KeyWithEventType) error {
 	//skip update or create if type is not mizar or network status is ready
 	key := eventKeyWithType.Key
+
+	if network.Spec.Type == mizarNetworkType && network.Status.Phase == v1.NetworkReady {
+		const subnetSuffix = "-subnet"
```

Comment:
const needs to be put in separate section, see line 56

Reply:
Yes. will do same thing for 3 const. Thanks!

Reply:
You have const in mizar-pod-controller as well

Reply:
I will make the changes, Thanks!

Reply:
made the changes.